Resources/Prototypes/_LostParadise/Entities/Objects/Weapons/Guns/Launchers/launchers.yml

```
@@ -0,0 +1,24 @@
+- type: entity
+  name: RB-4
+  parent: BaseWeaponLauncher
+  id: LPPWeaponCyborgLauncherRocket
+  description: Block of combat rockets for syndicate cyborg.
+  components:
+  - type: Sprite
+    sprite: _LostParadise/Objects/Weapons/Guns/Launchers/borg_launcher.rsi #_LostParadise\Objects\Weapons\Guns\Launchers
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)